app/views/layouts/application.html.erb

```diff
@@ -9,13 +9,13 @@
 <body>
 
 <div id='nav-bar'>
-  <h3><%= link_to 'X Overflow', root_path %></h3>
+  <h3 id='logo'><%= link_to 'X Overflow', root_path %></h3>
   <% if current_user %>
-    <p><%= link_to 'Ask', new_question_path %></p>
-    <p><%= link_to 'Logout', logout_path %></p>
+    <p id='link'><%= link_to 'Ask', new_question_path %></p>
```

Comment:
ids by nature are unique right? we don't want multiple ids that are the same value.  I think these would be better off classified as a p class = 'link'